### ISSUE TITLE
[Accessibility] Implement an API call to extends duration of forms

### ISSUE THREAD

**Issue by samuel-deal-tisseo** [keycloak/keycloak#44904]
### Description

Allow users to extends any forms expiration in order to to be more accessible for people with disabilities.

To do so we need:
* An API endpoint that can be called in ajax by the form when the user need to extends the duration
* 1n ftl variable to known the time limit in for so theme editors can write a javascript mathod to manage this time limit and call the API endpoint.
* Maybe a keyclock config boolean to enable/disable the API endpoint ?
* Maybe provide a new `max form duration` keycloak config to limit the form lifetime extension ?

### Value Proposition

The current proposition (I thing) this without compromising security, because:
* This feature can be off by default in keycloak config
* We don't need to extends token lifetimes
* We can keep the default `User-Initiated Action Lifespan`
* Keycloak interators can put an ip rate-limit on the API call (if the route is easily recognizable)

### Goals

Allow an keycloak theme implementation compatible with the following criteria:

* [WCAG 2.2, rule 2.2.1](https://www.w3.org/TR/WCAG22/#enough-time)
* [RGAA 4, rule 13.1 (in french)](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#13)

### Non-Goals

I don't know

### Notes

Context: We made an accessibility audit on our keycloak instance.

An issue I can't fix is the registration form (and other form like login, password change, etc...) expiration duration.

The audit state:
> Login and registration pages: User impact: significant
> When the session timeout expires, the login process must restart from the beginning.
> 
> This invalidates this criterion.
> The ideal solution would be to remove this time limit for both forms. If this is not possible, a system should be implemented to alert users to this time limit and then allow them to extend it.
> This point can be discussed in more detail during the audit feedback meeting.
